docs/src/pages/[platform]/components/breadcrumbs/react.mdx

```diff
@@ -0,0 +1,150 @@
+import {
+  BreadcrumbsThemeExample,
+  CustomSeparatorExample,
+  DefaultBreadcrumbsExample,
+  NextBreadcrumbsExample,
+  BreadcrumbsStyleExample,
+  CustomSeparatorIconExample,
+  BreadcrumbsArrayExample,
+} from './examples'
+
+import { ComponentStyleDisplay } from '@/components/ComponentStyleDisplay'
+import { Example, ExampleCode } from '@/components/Example'
+import ThemeExample from '@/components/ThemeExample.mdx'
+
+
```

Comment:
Didn't include a Demo since there is only 1 top level prop, `separator`, and most of the customization comes from the composition which can't really be done in our demo component